daml-lf/encoder/src/main/scala/com/digitalasset/daml/lf/archive/testing/EncodeV1.scala

```diff
@@ -178,7 +176,7 @@ private[digitalasset] class EncodeV1(val minor: LanguageMinorVersion) {
           builder.setCon(
             PLF.Type.Con.newBuilder().setTycon(tycon).accumulateLeft(args)(_ addArgs _))
         case TBuiltin(bType) =>
-          if (bType == BTArrow && V1.minorVersionOrdering.lteq(minor, "0")) {
+          if (bType == BTArrow && LV.ordering.lt(version, LV.v1_1)) {
```

Comment:
This changed from version `1.0` to version `1.1`.

Reply:
I change in the same time `lteq` by `lt`. I think it is more consistent with the `assertSince` we use in lot of other place.

Reply:
Right. I didn't notice that part. Whatever makes it more consistent with other checks works for me.